From 13f58940d587b74941c956456e500e18fffc980d Mon Sep 17 00:00:00 2001
From: Dan Harkins <dharkins@lounge.org>
Date: Sat, 11 Feb 2012 12:17:58 +0200
Subject: [PATCH 31/58] EAP-pwd: Describe build option for EAP-pwd

Signed-hostap: Dan Harkins <dharkins@lounge.org>
(cherry picked from commit 27c9d333d4bfab2b3ac377e5a7920e14a0fc6fe9)
---
 hostapd/defconfig        |    3 +++
 wpa_supplicant/defconfig |    3 +++
 2 files changed, 6 insertions(+)

--- a/hostapd/defconfig
+++ b/hostapd/defconfig
@@ -84,6 +84,9 @@ CONFIG_EAP_TTLS=y
 # EAP-PSK for the integrated EAP server (this is _not_ needed for WPA-PSK)
 #CONFIG_EAP_PSK=y
 
+# EAP-pwd for the integrated EAP server (secure authentication with a password)
+#CONFIG_EAP_PWD=y
+
 # EAP-SAKE for the integrated EAP server
 #CONFIG_EAP_SAKE=y
 
--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -165,6 +165,9 @@ CONFIG_EAP_OTP=y
 # EAP-PSK (experimental; this is _not_ needed for WPA-PSK)
 #CONFIG_EAP_PSK=y
 
+# EAP-pwd (secure authentication using only a password)
+#CONFIG_EAP_PWD=y
+
 # EAP-PAX
 #CONFIG_EAP_PAX=y
 
